microprofile/WFLY-17137-microprofile-jakarta-ee-10.adoc

```diff
@@ -0,0 +1,185 @@
+= MicroProfile aligned with Jakarta EE 10
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+This issue is about upgrading to most parts of MicroProfile Platform 6, which is aligned with Jakarta EE 10. 'most' is used since a decision has been made to not include the new MicroProfile Metrics version, as the inclusion of MicroMeter tracked by https://issues.redhat.com/browse/EAP7-1686[EAP7-1686] provides our observability strategy.
```

Comment:
```suggestion
This issue is about upgrading to most parts of MicroProfile Platform 6, which is aligned with Jakarta EE 10. 'most' is used since a decision has been made to not include the new MicroProfile Metrics version, as the inclusion of Micrometer tracked by https://issues.redhat.com/browse/EAP7-1686[EAP7-1686] provides our observability strategy.
```

Reply:
https://issues.redhat.com/browse/WFLY-14947 provides the metrics aspect of WildFly's observability strategy. :-)

Although @jasondlee should probably file a new JIRA for getting Micrometer into std WF, as right now WFLY-14947 is being used to track getting it into WFP 27.

Reply:
Filed: https://issues.redhat.com/browse/WFLY-17144

Side note: there is no component for "Micrometer". :)